webui/react/src/pages/JobQueue/JobQueue.tsx

```diff
@@ -68,7 +68,9 @@ const JobQueue: React.FC<Props> = ({ selectedRp, jobState }) => {
   const [pageState, setPageState] = useState<{ isLoading: boolean }>({ isLoading: true });
   const pageRef = useRef<HTMLElement>(null);
 
-  const { settings, updateSettings } = useSettings<Settings>(settingsConfig(jobState));
+  const { settings, updateSettings } = useSettings<Settings>(
```

Comment:
Incidental bug that was uncovered due to this PR. The config passed to `useSettings` needs to be a stable reference.